app/org/sagebionetworks/bridge/validators/SurveyValidator.java

```diff
@@ -146,10 +159,13 @@ private void validateRules(Errors errors, List<SurveyElement> elements) {
             SurveyElement element = elements.get(i);
             if (element instanceof SurveyQuestion) {
                 for (SurveyRule rule : ((SurveyQuestion)element).getConstraints().getRules()) {
-                    if (!alreadySeenIdentifiers.contains(rule.getSkipToTarget())) {
-                        errors.pushNestedPath("elements["+i+"]");
-                        rejectField(errors, "rule", "has a skipTo identifier that doesn't exist: %s", rule.getSkipToTarget());
-                        errors.popNestedPath();
+                    // This validation only applies to skipTo target rules.
+                    if (rule.getSkipToTarget() != null) {
+                        if (!alreadySeenIdentifiers.contains(rule.getSkipToTarget())) {
+                            errors.pushNestedPath("elements["+i+"]");
+                            rejectField(errors, "rule", "has a skipTo identifier that doesn't exist: %s", rule.getSkipToTarget());
+                            errors.popNestedPath();
+                        }
```

Comment:
Similarly, how hard would it be to add a test for a non-existent skip-to target?